src/views/v2/rsvp/ari/form.php

```diff
@@ -4,35 +4,37 @@
  * ARI Form
  *
  * Override this template in your own theme by creating a file at:
- * [your-theme]/tribe/tickets/v2/rsvp/ari/sidebar.php
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form.php
  *
  * See more documentation about our Blocks Editor templating system.
  *
  * @link {INSERT_ARTICLE_LINK_HERE}
  *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ * @var WP_Post|int $post The post object or ID.
+ *
  * @since TBD
  *
  * @version TBD
  */
 
+$meta   = Tribe__Tickets_Plus__Main::instance()->meta();
+$fields = $meta->get_meta_fields_by_ticket( $rsvp->ID );
+
 ?>
 <div class="tribe-tickets__rsvp-ar-form">
 
 	<?php $this->template( 'v2/rsvp/ari/form/title', [ 'rsvp' => $rsvp ] ); ?>
 
-	<?php
-	$meta   = Tribe__Tickets_Plus__Main::instance()->meta();
-	$fields = $meta->get_meta_fields_by_ticket( $rsvp->ID );
-	?>
 	<div class="tribe-tickets__form">
 		<?php foreach ( $fields as $field ) : ?>
 			<?php
 				$args = [
-					'event_id'   => $event_id,
-					'ticket'     => $ticket,
+					'event_id'   => $post_id,
```

Comment:
Should we strive to keep them all post_id, like in the field templates themselves too?

Reply:
You mean the var name? I used `event_id` for them because I'm componetizing them and they were already using `event_id`